astropy/units/tests/test_typestubs_for_units.py

```diff
@@ -1,28 +1,30 @@
-import sys
+import subprocess
 import textwrap
 from pathlib import Path
 
-from astropy.units.scripts import typestubs_for_units
+import astropy.units as u
 
 
-def test_typestubs_for_units_script(tmp_path: Path, monkeypatch):
+def test_typestubs_installed():
+    spot_check_stubs(Path(u.__file__).parent)
```

Comment:
Note that our tests cover editable installs: https://github.com/astropy/astropy/blob/1ba18e8ace31b168d284a4104a4ed04b94ef9a44/.github/workflows/ci_workflows.yml#L72-L81